### PR TITLE
updated package.json

### PR DESCRIPTION
**Pull request by arpit1997**
reference issue #4

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,19 @@
+0 info it worked if it ends with ok
```
Comment (freekmurze) [inline npm-debug.log]: Please remove this file.

**Comment by arpit1997** [comment]
Left the debug logs by mistake. Sorry for that.

**Comment by sebastiandedeyne** [comment]
Slightly confused, how does this reference #4?

**Comment by arpit1997** [comment]
The issue #4 asks for updating package.json so I updated the dependencies in the package.

**Comment by sebastiandedeyne** [comment]
Oh, actually issue #4 means to update the `peerDepencies` that this package installs in the packages `package.json` file :)

This is welcome too though, thanks!

**Comment by arpit1997** [comment]
Thanks for merging. I will do the issue #4 too don't worry.

**Pull request merged**